docs/source/features/communication_overlap.rst

```diff
@@ -0,0 +1,63 @@
+Communication Overlap
+====================
+
+Data-parallel communication overlap
+-----------------------------------
+
+NeMo supports the overlap of the data-parallel (DP) communications with the computations in LLM training.
+NeMo features Distributed Optimizer that distributes optimizer states and the high precision master parameters across GPUs and this introduces two types of data-parallel communications; reduce-scatter of gradients and all-gather of updated parameters.
+The DP communication is chunked by the granularity of a Transformer layer and overlap each communication chunk with computation.
```

Comment:
The DP communication is chunked by the granularity of a Transformer layer and overlaps each communication chunk with computation.